src/Neo.CLI/config.mainnet.json

```diff
@@ -20,7 +20,8 @@
       "Path": "",
       "Password": "",
       "IsActive": false
-    }
+    },
+    "Plugins": ["LevelDBStore"]
```

Comment:
If we define the plugins in the config, we should not try to find them in the folder

Reply:
what?

Reply:
> And i dont understand your we should not try to find them in the folder.

Currently the plugins are "installed" if they are present in the folder

Reply:
exactly, the install command will check if they exists, if so, the install command will return directly.

Reply:
People may get confused when they see this. Because we don't update `Plugins` list?